### PR TITLE
setup for africa ows cfg testing

### PR DESCRIPTION
**Pull request by pindge**
- setup for  africa ows cfg testing
- split large ows_cfg file to multi files
- add `inventory.json` to benchmark layers and styles

### CODE REVIEW

File: services/ows_refactored/prod/landsat/ows_s2_cfg.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+from prod.landsat.style_ls_cfg import styles_s2_list
```
Comment (SpacemanPaul) [inline services/ows_refactored/prod/landsat/ows_s2_cfg.py:1]: Why is Sentinel-2 sitting under landsat in the hierarchy?

**Review by SpacemanPaul — APPROVED** [review]
Note comments as things that need doing soon.

File: services/ows_refactored/prod/landsat/ows_s2_cfg.py (hunk 1 above)
Comment (pindge) [reply services/ows_refactored/prod/landsat/ows_s2_cfg.py:1]: they share the same styles, i dont know why

File: services/ows_refactored/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# OWS CFG refactored
+Goal: Split mega single file ows_cfg.py into multiple manageable/maintainable config files.
+
```
[the remark this answers is not in the document]
Comment (pindge) [reply services/ows_refactored/README.md:3]: added some words in readme

File: services/ows_refactored/prod/landsat/ows_s2_cfg.py (hunk 1 above)
Comment (pindge) [reply services/ows_refactored/prod/landsat/ows_s2_cfg.py]: renamed to `surface_reflectance`

**Review by fangfy — APPROVED** [review]
This looks great! so much easier to debug and update in future. Are we going to have a dev-af next?

**Comment by pindge** [comment]
> This looks great! so much easier to debug and update in future. Are we going to have a dev-af next?

next pr

**Pull request merged**